internal/storage/database/repositories/auth_db/auth_db.go

```diff
@@ -0,0 +1,46 @@
+package auth_db
+
+import (
+	"booker/internal/storage/database"
+	"context"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"log/slog"
+	"strconv"
+)
+
+type Tokens struct {
+	access_token  string
+	refresh_token string
+}
+
+type TokensRepository interface {
+	DbPutTokens(ctx context.Context, userId int64, accessToken string, refreshToken string) error
+}
+type TokensRepositoryImpl struct {
+	db  *pgxpool.Pool
+	log *slog.Logger
+}
+
+func NewTokensRepositoryImpl(db *pgxpool.Pool, log *slog.Logger) *TokensRepositoryImpl {
+	return &TokensRepositoryImpl{
+		db:  db,
+		log: log,
+	}
+}
+
+func (r *TokensRepositoryImpl) DbPutTokens(ctx context.Context, userId int64, accessToken string, refreshToken string) error {
```

Comment:
```suggestion
func (r *TokensRepositoryImpl) DbPutTokens(ctx context.Context, userId int64, tokens Tokens) error {
```
ты можешь передать объект целиком и не разделять его на отдельные передаваемые строки